pyproject.toml

```diff
@@ -248,21 +248,21 @@ dependencies = [
 ]
 
 [tool.hatch.envs.default.scripts]
-tests = "pytest -n auto {args}"
+tests = "pytest {args}"
```

Comment:
Do we no longer need `pytest-xdist`?

Reply:
how many cores are we using if we don't specify this option? do we still need `pytest-xdist`?